### ISSUE TITLE
[BITV] 9.1.4.3/1.1 - The error message "Incorrect username or password" does not meet the contrast requirements for text - it is 4.3:1. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36949]
- [x] https://github.com/nextcloud/server/pull/38211
- [x] https://github.com/nextcloud/nextcloud-vue/pull/4188
- [x] https://github.com/nextcloud/nextcloud-vue/pull/4171
- [x] Bump

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-6fe46b78da-v1-n1

</details>

**Comment by Pytal** [comment]
From investigations this is done in NcInputField `helperText` which is displayed with `--color-error` on error state but also `--color-success` on success state

Therefore both these variables will need to be updated to meet the minimum 4.5:1 contrast requirements and checked on light and dark backgrounds

**Comment by Pytal** [comment]
Since https://github.com/nextcloud/server/pull/38211 is in for target master 28, https://github.com/nextcloud/nextcloud-vue/pull/4188 can only get into server once v8 final is released
